sources/osg/CullSettings.js

```diff
@@ -9,12 +9,17 @@ define( [], function () {
         this.bbCornerFar = ( lookVector[ 0 ] >= 0 ? 1 : 0 ) | ( lookVector[ 1 ] >= 0 ? 2 : 0 ) | ( lookVector[ 2 ] >= 0 ? 4 : 0 );
         this.bbCornerNear = ( ~this.bbCornerFar ) & 7;
         /*jshint bitwise: true */
+        this._enableFrustumCulling = false;
+
+        this._settingsSource = this;
```

Comment:
It's a way to handle the enableFrustumCulling at global scale.
(as in url parameter, overriding any further cullsettings which are per camera.)

The override check is here; https://github.com/cedricpinson/osgjs/pull/237/files#diff-ece7c42cf3b8e9cbd8392b2b82e1e6ebR274

Will add codesplaining